CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # Psammead Changelog
 
-
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.1.4   | [PR#446](https://github.com/BBC/psammead/pull/423) Update Storybook to version 5.0.6 |
```

Comment:
There is a `CHANGELOG.md` file per package. e.g. https://github.com/bbc/psammead/blob/latest/packages/components/psammead-caption/CHANGELOG.md as well as one for the Psammead repository as a whole.  

Since this PR is changing only the Psammead Caption package, you only need to update the specific Psammead Caption Changelog file. 

Also, when adding a line in the Changelog, it's good to add a description of the changes you've made, that'll be help someone using the package understand what's updated.  So in this case, ensuring nested `<i>` italic elements will be styled as not italic.

Reply:
oh wow sorry! I thought this looked like the wrong place.. and I have no idea why I didn't change that line to reflect the changes 😂  but thanks! I'll move it